Turf/Turf.swift

```diff
@@ -285,3 +285,65 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var radius:Float = 6378137
```

Comment:
Let's rename this to `equatorialRadius` for clarity and move it to the top where `metersPerRadian` is.